### ISSUE TITLE
Crash when writing to SD card on LG G5

### ISSUE THREAD

**Issue by syy819** [TeamAmaze/AmazeFileManager#1688]
Been using Amaze for years on my G5 but recently it started crash writing operations to external storage. Read works fine. I can copy from external to internal but not the other way round or copy/cut anything within external storage. The built-in LG file manager seems to be working perfectly and so does the ES File Manager.

Any help would be appreciated since Amaze is a very useful file manager to me! Thanks in advance.

**Comment by EmmanuelMess** [comment]
Which Android version?

**Comment by syy819** [comment]
Android 8 Oreo. Details below:
![Screenshot_2019-09-30-00-02-14](https://user-images.githubusercontent.com/55671090/65847020-31744580-e316-11e9-82ee-90f7b8c20d20.png)

**Comment by VishalNehra** [comment]
Please also mention Amaze version that you use. And logs would help a lot.

**Comment by syy819** [comment]
I'm using v3.3.4.

I just solved this issue by trying to create a file in the root of my external storage. It finally prompted me to grant write access. After doing so, all the copy/cut-paste operations run perfectly fine.

I think what happened is after a certain update, the app needed to regain write access for external memory. However, instead asking for write access, everytime a write request is initiated, it crashed. I've tried reinstalling the app with no luck. It finally prompted and regained write access.

It is solved now so I can't repeat the crash and thus can't get any logs. Sorry about that. I didn't have logcat installed before so didn't catch the crash back then.
